app/views/collections/show.rb

```diff
@@ -24,7 +25,7 @@ def head_meta
           { meta_property: 'og:url', content: collection_url(@collection.key) }
         ]
         head_meta << { meta_property: 'og:title', content: title } unless title.nil?
-        head_meta << { meta_property: 'og:image', content: URI.join(root_url, hero[:hero_image]) } unless hero.nil?
+        head_meta << { meta_property: 'og:image', content: @landing_page.og_image } unless @landing_page.og_image.blank?
```

Comment:
This potentially results in all of the logic in `Page::Landing#og_image_from_promo` and `Page::Landing#og_image_from_hero` being run twice.

Could we avoid that by either storing `@og_image` on `Page::Landing` after `#og_image` is first called, or storing in a local variable in `Collections::Show#head_meta`?